tests/unit/asio_service_stream_test.cxx

```diff
@@ -29,57 +29,178 @@ using namespace raft_functional_common;
 namespace asio_service_stream_test {
     const std::string TEST_MSG = "stream-test-msg-str";
 
+    class stream_statistic {
```

Comment:
I just realized that you put indent inside `namespace`. As you can see in other tests, no indent is needed. Please remove it.